charts/prometheus-adapter/templates/deployment.yaml

```diff
@@ -55,6 +55,7 @@ spec:
         - --logtostderr=true
         - --prometheus-url={{ tpl .Values.prometheus.url . }}{{ if .Values.prometheus.port }}:{{ .Values.prometheus.port }}{{end}}{{ .Values.prometheus.path }}
         - --metrics-relist-interval={{ .Values.metricsRelistInterval }}
+        - --metrics-max-age={{ .Values.metricsMaxAge }}
```

Comment:
Why is this change necessary? Can't you just use `extraArguments` to pass `--metrics-max-age`?

Reply:
Sorry, using extraArguments, is also possible